Automattic/sensei
# Fix timed quiz still working for students even after time ends

Resolves https://github.com/Automattic/sensei-pro/issues/2606

## Proposed Changes

In timed quizzes, when time is up, the frontend code [clicks the completion button](https://github.com/Automattic/sensei-pro/blob/8c1c24bb349764c439afcc77c40cea92bad0826e/modules/advanced-quiz/assets/blocks/quiz-timer/frontend.js#L8-L13) to submit the quiz. But in the cases where the quiz was paginated, completion button only got rendered on the last page. So if the student was not on the last page, the quiz didn't get submitted, and the student was still able to answer.

To fix that, we're rendering the button even in the middle pages so that the completion function still works, but keeping it hidden in those cases so that UI doesn't break or change from before.

## Testing Instructions


1. Create a Course with a lesson that has a quiz with many questions
2. Make sure you have Sensei Pro enabled
3. From the quiz settings, set pagination to `Multi Page` and set `per page` value to 1 or a small number to generate many pages
4. Also enable `Quiz Timer` option, set a small time
5. Now take the quiz as a student, be on any page other than the last page
6. Make sure when time is up, the quiz gets submitted
7. Now take the quiz as another student, finish the quiz normally and make sure it works
8. Now take the quiz as a student for the third time, stay on the last page of the quiz without submitting, make sure the quiz gets submitted automatically when the time is up.

## Pre-Merge Checklist
